router/pkg/trace/art.go

```diff
@@ -0,0 +1,110 @@
+package trace
+
+import (
```

Comment:
trace package only contains files about OTEL tracing. Could you put this in e.g ART package?

Reply:
Done